server/applicationset/applicationset.proto

```diff
@@ -28,6 +29,13 @@ message ApplicationSetListQuery {
 }
 
 
+// ApplicationSetEventsQuery is a query for applicationset resource events
+message ApplicationSetResourceEventsQuery {
+	string name = 1;
+	string appsetNamespace = 5;
```

Comment:
I would expect the number to increase sequentially. Any reason why it is 5? Perhaps it should be added as a comment if there is a valid reason.
```suggestion
	string appsetNamespace = 2;
```

Reply:
@alexymantha could you comment on that?

Reply:
@agaudreault is correct. This seems to be a bad copy/paste. Attributes in protobuf messages should have a sequential number and once assigned, it should never change. This guarantees API backward compatibility.

Reply:
Yep it was a bad copy/paste. I fixed it on my branch @reggie-k if you want to merge